tests/pynfs/install.pm

```diff
@@ -0,0 +1,56 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Install pynfs
+# Maintainer: Yong Sun <yosun@suse.com>
+package install;
+
+use 5.018;
+use strict;
+use warnings;
+use base 'opensusebasetest';
+use utils;
+use testapi;
+
+sub install_dependencies {
+    my @deps = qw(
+      git
+      krb5-devel
+      python3-devel
+      swig
+      python3-gssapi
+      python3-ply
+      nfs-client
+      nfs-kernel-server
+    );
+    zypper_call('in ' . join(' ', @deps));
+}
+
+sub install_pynfs {
+    install_dependencies;
+    assert_script_run('git clone -q --depth 1 git://git.linux-nfs.org/projects/bfields/pynfs.git');
+    assert_script_run('cd ./pynfs && ./setup.py build && ./setup.py build_ext --inplace');
+}
```

Comment:
Could we have `record_info()` with output of `git log -1 --pretty=format:"git-%h"` (I suppose `git describe` will not be working) similar to `install_ltp.pm`?

Reply:
Added, and the folder changes in previous commit which I forgot to sync to generic_report.pm...new VR: http://10.67.133.133/tests/49